ivy/functional/frontends/paddle/nn/functional/loss.py

```diff
@@ -1 +1,84 @@
 # local
+import ivy
+import ivy.functional.frontends.torch as torch_frontend
+from ivy.functional.frontends.torch.func_wrapper import to_ivy_arrays_and_back
+from ivy.func_wrapper import with_supported_dtypes
+
+
+def norm(input, axis):
+    return ivy.sqrt(ivy.sum(ivy.square(input), axis=axis))
+
+
+def pairwise_distance(x1, x2, *, p=2.0, eps=1e-06, keepdim=False):
+    x1, x2 = torch_frontend.promote_types_of_torch_inputs(x1, x2)
+    x1_dim = len(x1.shape)
+    x2_dim = len(x2.shape)
+    if x1_dim > x2_dim:
+        output_dim = x1_dim
+    else:
+        output_dim = x2_dim
+
+    return ivy.vector_norm(x1 - x2 + eps, ord=p, axis=output_dim - 1, keepdims=keepdim)
+
+
+def cosine_similarity(x1, x2):
```

Comment:
And this

Reply:
Hello @hirwa-nshuti  Below are functions in torch FrontEnd  which does not have test cases and same functions are needed for paddle FrontEnd also. 
Should I write test cases for and make a proper PR ? As they will be helpful in Paddle Loss functions also.... and might be upcoming frameworks? 
norm, pairwise_distance, cosine_similarity, _get_reduction_func, _legacy_get_string, _get_reduction_method, _get_reduction_string, _apply_reduction

Reply:
@hirwa-nshuti  I think, I will keep them as it is... because the same implementation is done in Torch FrontEnd and I think it has some importance but authors, but I am not getting that certain doubt in docs.